tests/lib/rules/no-fallthrough.js

```diff
@@ -133,6 +141,14 @@ ruleTester.run("no-fallthrough", rule, {
             code: "switch(foo) { case 0:\n\n default: b() }",
             errors: errorsDefault
         },
+        {
+            code: "switch(foo) { case 0: {} default: b() }",
+            errors: errorsDefault
+        },
+        {
+            code: "switch(foo) { case 0: { /* comment */ } default: b() }",
+            errors: errorsDefault
+        },
```

Comment:
It would be nice to also add tests where the block isn't the only statement in the case, and a test where the only statement isn't a block, e.g.:

```js
{
    code: "switch(foo) { case 0: a(); { /* falls through */ } default: b() }",
    errors: errorsDefault
},
{
    code: "switch(foo) { case 0: { /* falls through */ } a(); default: b() }",
    errors: errorsDefault
},
{
    code: "switch(foo) { case 0: if (a) { /* falls through */ } default: b() }",
    errors: errorsDefault
}
```

Reply:
Done.